src/app/helptext/network/interfaces/interfaces-form.ts

```diff
@@ -60,4 +56,65 @@ alias_netmaskbit6_tooltip : T('Select the prefix length used on the network.'),
 delete_placeholder6: T('Delete'),
 delete_tooltip6: T('Set to delete this alias.'),
 
+bridge_members_placeholder: T('Bridge Members'),
+bridge_members_tooltip: T(''),
```

Comment:
From FreeBSD documentation:

"Network interfaces to include in the bridge."

Reply:
@wblock @william-gr this seems like a resonable tooltip label to me, what do you say?

Reply:
yes